### PR TITLE
Code style fixes

### PR DESCRIPTION
**Pull request by damasco**

### CODE REVIEW

File: tests/_bootstrap.php [hunk 1]
```diff
@@ -2,5 +2,5 @@
 define('YII_ENV', 'test');
 defined('YII_DEBUG') or define('YII_DEBUG', true);
 
-require_once(__DIR__ . '/../vendor/yiisoft/yii2/Yii.php');
-require __DIR__ .'/../vendor/autoload.php';
\ No newline at end of file
+require_once __DIR__ . '/../vendor/yiisoft/yii2/Yii.php';
+require_once __DIR__ . '/../vendor/autoload.php';
```
Comment (samdark) [inline tests/_bootstrap.php]: Why?

File: views/site/login.php [hunk 2]
```diff
@@ -1,8 +1,10 @@
 <?php
 
-/* @var $this yii\web\View */
-/* @var $form yii\bootstrap\ActiveForm */
-/* @var $model app\models\LoginForm */
+/**
+ * @var \yii\web\View $this
+ * @var \yii\bootstrap\ActiveForm $form
+ * @var \app\models\LoginForm $model
+ */
```
Comment (rob006) [inline views/site/login.php]: Why did you change that? `/* @var $this yii\web\View */` has better IDE support.

Comment (damasco) [inline views/site/login.php]: https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc.md

File: tests/_bootstrap.php (hunk 1 above)
Comment (damasco) [inline tests/_bootstrap.php]: Redundantly?

https://github.com/yiisoft/yii2-app-advanced/blob/master/backend/tests/_bootstrap.php

File: tests/_bootstrap.php [hunk 3]
```diff
@@ -2,5 +2,5 @@
 define('YII_ENV', 'test');
 defined('YII_DEBUG') or define('YII_DEBUG', true);
 
-require_once(__DIR__ . '/../vendor/yiisoft/yii2/Yii.php');
-require __DIR__ .'/../vendor/autoload.php';
\ No newline at end of file
+require_once __DIR__ . '/../vendor/autoload.php';
+require_once __DIR__ . '/../vendor/yiisoft/yii2/Yii.php';
```
Comment (cebe) [inline tests/_bootstrap.php]: no need for require_once, including this file twice should be an error.

File: views/site/login.php (hunk 2 above)
Comment (cebe) [inline views/site/login.php]: its a proposal, don't take it too serious. We'll keep the style that has better IDE support until there is a standard that we can refer to when asking IDEs to support something.

**Comment by samdark** [comment]
@damasco do you plan to work on the pull request more?

**Comment by cebe** [comment]
most of this seems to be solved by https://github.com/yiisoft/yii2-app-basic/commit/4dc90946e9a652b9bb7fc381c53afc548e7ac5d5, thanks!
